packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -76,50 +45,92 @@ const ActiveVisitsTable = (props) => {
   const [activeVisits, setActiveVisits] = useState<ActiveVisitRow[]>([]);
   const [searchString, setSearchString] = useState('');
 
-  const searchResults = useMemo(() => {
-    if (searchString && searchString.trim() !== '') {
-      const search = searchString.toLowerCase();
-      return activeVisits.filter((activeVisitRow) =>
-        Object.keys(activeVisitRow).some((header) => {
-          if (header === 'patientUuid') {
-            return false;
-          }
-          return `${activeVisitRow[header]}`.toLowerCase().includes(search);
-        }),
-      );
-    } else {
-      return activeVisits;
-    }
-  }, [searchString, activeVisits]);
-  const { goTo, currentPage, results } = usePagination(searchResults, currentPageSize);
+  const headerData = useMemo(
+    () => [
+      {
+        id: 0,
+        header: t('visitStartTime', 'Visit Time'),
+        key: 'visitStartTime',
+      },
+      {
+        id: 1,
+        header: t('IDNumber', 'ID Number'),
+        key: 'IDNumber',
+      },
+      {
+        id: 2,
+        header: t('name', 'Name'),
+        key: 'name',
+      },
+      {
+        id: 3,
+        header: t('gender', 'Gender'),
+        key: 'gender',
+      },
+      {
+        id: 4,
+        header: t('age', 'Age'),
+        key: 'age',
+      },
+      {
+        id: 5,
+        header: t('visitType', 'Visit Type'),
+        key: 'visitType',
+      },
+    ],
+    [t],
+  );
 
   useEffect(() => {
-    const activeVisits = fetchActiveVisits().subscribe((data) => {
+    fetchActiveVisits().then(({ data }) => {
```

Comment:
Don't forget to pass in an `AbortController` which you cancel when the effect is disposed. Important to **not** set state when the component was unmounted.

Reply:
Oh okay! I'll make the change asap